src/MdDocs.Common/Grynwald.MdDocs.Common.csproj

```diff
@@ -12,20 +12,11 @@
 
   <ItemGroup>
     <PackageReference Include="Grynwald.MarkdownGenerator" Version="2.5.34" />
+    <PackageReference Include="Grynwald.Utilities.Configuration" Version="1.6.122" />
```

Comment:
I couldn't get the code to compile without adding this reference

Reply:
I've reverted your build changes, since including this package (in particular the newer versions of `Microsoft.Extensions.Configuration.*`) has caused some issue the MdDocs MSBuild task in the past.

To build locally, please clone using the `--recursive` flag since the Utilities assembly is included as git submodule)